src/Schema/Grammar.php

```diff
@@ -631,6 +697,19 @@ protected function typeBoolean(Fluent $column)
         return 'bool';
     }
 
+    /**
+     * @param Blueprint $blueprint
+     * @param Fluent<string, mixed> $column
+     * @return string|null
+     */
+    protected function modifyUseSequence(Blueprint $blueprint, Fluent $column): ?string
+    {
+        if (isset($column->useSequence)) {
+            return " default (get_next_sequence_value(sequence {$column->useSequence}))";
```

Comment:
It would be better to wrap like other methods?

```suggestion
            return " default (get_next_sequence_value(sequence {$this->wrap($column->useSequence)}))";
```

Reply:
Fixed.